# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/arcticfox/hotfix-icu4c-std.patch
# Copyright (C) 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- Arctic-Fox-45.3/build/moz.configure/toolchain.configure.orig	2025-03-13 14:33:35.000000000 +0100
+++ Arctic-Fox-45.3/build/moz.configure/toolchain.configure	2025-04-16 22:00:54.715873875 +0200
@@ -321,9 +321,9 @@
     # Note: MSVC, while supporting C++11, still reports 199711L for __cplusplus.
     # Note: this is a strict version check because we used to always add
     # -std=gnu++11.
-    if info.language == 'C++' and info.language_version != 201103:
-        if info.type in ('clang-cl', 'clang', 'gcc'):
-            append_flag('-std=gnu++11')
+    #if info.language == 'C++' and info.language_version != 201404:
+    #    if info.type in ('clang-cl', 'clang', 'gcc'):
+    #        append_flag('-std=gnu++17')
 
     # We force clang-cl to emulate Visual C++ 2013 Update 3 with fallback to
     # cl.exe.
